ner/model.py

```diff
@@ -1,10 +1,29 @@
 import pandas as pd
 from sqlalchemy.orm import sessionmaker
+import logging
+import os
+import spacy
 
-from ner import settings
-from ner.db import DB
+#from ner import settings
+#from ner.db import DB
 
-print(settings.env)
+#print(settings.env)
+
+def extract_entities(input_text):
+    # get latest model
+    models_dir = os.path.join('..', 'Models') if __name__ == '__main__' else 'Models'
+    latest_model = max([d for d in os.listdir(models_dir)])
```

Comment:
this is the same as
```suggestion
    latest_model = max(os.listdir(models_dir))
```
can you remind me again how the models are named? and is this a reliable way of picking the latest model?

Reply:
The models are named thus: 'model_YYYYMMDD-HHMMSS', so I think it should work?

Reply:
yes, that should work. My small suggestion still stands since the output of `listdir` is a list of a strings.